### PR TITLE
docs(applications): explain how to add user ssh key

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

File: src/applications/using-buildpacks.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 Deis supports deploying applications via [Heroku Buildpacks][]. Buildpacks are useful if you're interested in following Heroku's best practices for building applications or if you are deploying an application that already runs on Heroku.
 
+## Add SSH Key
+ 
+For **Buildpack** based application deploys via `git push`, Deis Workflow identifies users via SSH keys. SSH keys are pushed to the platform and must be unique to each user.
```
Comment (arschles) [inline src/applications/using-buildpacks.md]: shouldn't this also include Dockerfile based application deploys?

File: src/applications/using-buildpacks.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+
+How to generate the SSH key check it [here](../users/ssh-keys.md/#generate-an-ssh-key).
```
Comment (arschles) [inline src/applications/using-buildpacks.md]: Can this say `See [this document](../users/ssh-keys.md/#generate-an-ssh-key) for instructions on how to generate an SSH key`?

File: src/applications/using-buildpacks.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
...
+
+Run `deis keys:add` to upload your SSH key to Deis Workflow.
+
+```
+$ deis keys:add ~/.ssh/id_deis.pub
+Uploading id_deis.pub to deis... done
+```
```
Comment (arschles) [inline src/applications/using-buildpacks.md]: it seems like there are a few step-by-step instructions here. Is it worth putting them into a bulleted or ordered list?

File: src/applications/using-buildpacks.md (hunk 1 above)
Comment (arschles) [inline src/applications/using-buildpacks.md]: Nevermind, I just read the below Dockerfile docs

File: src/applications/using-dockerfiles.md [hunk 4]
```diff
@@ -3,6 +3,21 @@
 Deis supports deploying applications via Dockerfiles.  A [Dockerfile][] automates the steps for crafting a [Docker Image][].
 Dockerfiles are incredibly powerful but require some extra work to define your exact application runtime environment.
 
+## Add SSH Key
+ 
+For **Dockerfile** based application deploys via `git push`, Deis Workflow identifies users via SSH keys. SSH keys are pushed to the platform and must be unique to each user.
+
+How to generate the SSH key check it [here](../users/ssh-keys.md/#generate-an-ssh-key).
```
Comment (arschles) [inline src/applications/using-dockerfiles.md]: same here regarding the rewording (mentioned above)

File: src/applications/using-dockerfiles.md [hunk 5]
```diff
@@ -3,6 +3,21 @@
...
+
+Run `deis keys:add` to upload your SSH key to Deis Workflow.
+
+```
+$ deis keys:add ~/.ssh/id_deis.pub
+Uploading id_deis.pub to deis... done
+```
```
Comment (arschles) [inline src/applications/using-dockerfiles.md]: same here regarding the step-by-step instructions (mentioned above)

**Comment by mboersma** [comment]
This looks like a helpful addition to the docs. I'll review it after the existing comments are addressed.

**Comment by helgi** [comment]
@rimusz are you going to be able to look at the comments and update the docs?

**Comment by rimusz** [comment]
closing if favour of #418

**Pull request closed without merging**